ios/FluentUI/Pill Button Bar/PillButtonStyle.swift

```diff
@@ -24,141 +24,130 @@ public extension PillButton {
     // MARK: normal state
 
     @objc(normalBackgroundColorForWindow:ForStyle:)
-    static func normalBackgroundColor(for window: UIWindow, for style: PillButtonStyle) -> UIColor {
-        let defaultColor = UIColor(light: Colors.surfaceTertiary, dark: Colors.surfaceSecondary)
+    static func normalBackgroundColor(for fluentTheme: FluentTheme, for style: PillButtonStyle) -> UIColor {
```

Comment:
I know this was an existing behavior, but while we're updating these static color APIs can we get rid of the double `for:` labels? A signature like `normalBackgroundColor(for:for:)` is a bit confusing. Maybe dropping one or both of the `for` labels?

Same for the functions below, unless you think it'd be too disruptive to change right now.

Reply:
This will all be replaced when we update with tokens anyway, so it may not be worth it to change now.